packages/launch/preset/_data/site.cjs

```diff
@@ -9,6 +9,11 @@ module.exports = function () {
         name: 'GitHub',
         url: 'https://github.com/modernweb-dev/rocket',
       },
+      {
+        name: 'Slack',
+        url:
+          'https://join.slack.com/t/lit-and-friends/shared_invite/zt-llwznvsy-LZwT13R66gOgnrg12PUGqw',
```

Comment:
that should probably be in `docs/_data` as it's specific to rocket right?

we probably also need to hide this on mobile (as more than one icon in the header will take too much space)... hmm or move the link to the footer? 🤔

Reply:
the link right above is also specific to rocket, no?